code/inc/gfx/element/gfx_vertex.hpp

```diff
@@ -0,0 +1,122 @@
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+
+* You should have received a copy of the GNU General Public License
+* along with this program.  If not, see <http://www.gnu.org/licenses/>.
+**************************************************************************************************/
+
+#pragma once
+
+/*=================================================================================================
+** 1.  REFERENCES
+**===============================================================================================*/
+
+/*=================================================================================================
+** 2.  INCLUDE FILES
+**===============================================================================================*/
+
+#include "gfx/gfx_common.hpp"
+
+/*=================================================================================================
+** 3.  DECLARATIONS
+**
+** 3.1 Macros
+**===============================================================================================*/
+
+/*=================================================================================================
+** 3.2 Types and Classes
+**===============================================================================================*/
+namespace GFX
+{
+    namespace Element
+    {
+        /**************************************************************************************************/
+        /**
+        * \par Details: 
+        */
+        class Vertex
+        {
+
+        public:
+            /*********************************/
+            // Public type definitions
+            /*********************************/
+
+            /*********************************/
+            // Public member variables
+            /*********************************/
+
+            glm::vec3 m_Position;
+            glm::vec3 m_Normal;
+
+            /*********************************/
+            // Constructors/Destructor
+            /*********************************/
+
+            /**************************************************************************************************/
+            /**
+            * \brief The default constructor for the Vertex.
+            */
+            Vertex(void);
+
+            /**************************************************************************************************/
+            /**
+            * \brief Initialize the vertex by string to be tokenized.
+            * 
+            * \param[in] tokens The tokens to initialize the vertex.
+            */
+            Vertex(const std::vector<std::string> tokens);
+
+            /**************************************************************************************************/
+            /**
+            * \brief 
+            * 
+            * \param[in] 
+            * 
+            * \retval 
+            */
```

Comment:
Fill in comment

Reply:
Fixed https://github.com/tdiego1/worldweaver/pull/19/commits/1a0df967e70f1752177ebca99e4618e4ec06c26d